.circleci/config.yml

```diff
@@ -108,7 +108,7 @@ parameters:
     default: 20.10.7
   api-go-version:
     type: string
-    default: "1.18"
+    default: "1.19"
```

Comment:
Even though doc says below; connect-go fails if go --version is not 1.19.
```
You'll need one of the [last two major releases](https://golang.org/doc/devel/release) of Go, with a minimum of Go 1.18. See Go's [Getting Started](https://golang.org/doc/install) guide for installation instructions.
```

Reply:
Bumping the Go version of the API package means anyone using the SDK on 1.18 will be unable to use it anymore. I know at some point it was a bit of a concern to force users to use recent versions of Go, not sure where we stand those days, but worth being extra loud about it if we end up merging this

Reply:
@gbouv have message the folks at buf to see what they say.. i agree with you;; will ping the team regarding this.